docs/_layouts/global.html

```diff
@@ -69,14 +69,14 @@
                             <a href="#" class="dropdown-toggle" data-toggle="dropdown">Programming Guides<b class="caret"></b></a>
                             <ul class="dropdown-menu">
                                 <li><a href="quick-start.html">Quick Start</a></li>
-                                <li><a href="programming-guide.html">Spark Programming Guide</a></li>
-                                <li class="divider"></li>
-                                <li><a href="streaming-programming-guide.html">Spark Streaming</a></li>
                                 <li><a href="sql-programming-guide.html">DataFrames, Datasets and SQL</a></li>
                                 <li><a href="structured-streaming-programming-guide.html">Structured Streaming</a></li>
                                 <li><a href="ml-guide.html">MLlib (Machine Learning)</a></li>
                                 <li><a href="graphx-programming-guide.html">GraphX (Graph Processing)</a></li>
                                 <li><a href="sparkr.html">SparkR (R on Spark)</a></li>
+                                <li class="divider"></li>
+                                <li><a href="rdd-programming-guide.html">RDDs, Accumulators, Broadcasts Vars</a></li>
```

Comment:
Actually, I think we probably still need to keep this after "Quick Start". The following sections inside this page are pretty important for most of Spark modules:

- Overview
- Linking with Spark
- Initializing Spark
- Using the Shell
- Passing Functions to Spark
- Understanding closures
- Local vs. cluster modes
- Deploying to a Cluster
- Launching Spark jobs from Java / Scala
- Unit Testing